art/estimators/object_detection/tensorflow_faster_rcnn.py

```diff
@@ -315,21 +322,25 @@ def _load_model(
 
         return obj_detection_model, predictions, losses, detections
 
-    def loss_gradient(self, x: np.ndarray, y: List[Dict[str, np.ndarray]], **kwargs) -> np.ndarray:
+    def loss_gradient(  # pylint: disable=W0221
+        self, x: np.ndarray, y: List[Dict[str, np.ndarray]], standardise_output: bool = False, **kwargs
+    ) -> np.ndarray:
         """
         Compute the gradient of the loss function w.r.t. `x`.
 
         :param x: Samples of shape (nb_samples, height, width, nb_channels).
-        :param y: A dictionary of target values. The fields of the dictionary are as follows:
-
-                  - `boxes`: A list of `nb_samples` size of 2-D tf.float32 tensors of shape [num_boxes, 4] containing
-                    coordinates of the groundtruth boxes. Groundtruth boxes are provided in [y_min, x_min, y_max, x_max]
-                    format and also assumed to be normalized as well as clipped relative to the image window with
-                    conditions y_min <= y_max and x_min <= x_max.
-                  - `labels`: A list of `nb_samples` size of 1-D tf.float32 tensors of shape [num_boxes] containing
-                    the class targets with the zero index assumed to map to the first non-background class.
-                  - `scores`: A list of `nb_samples` size of 1-D tf.float32 tensors of shape [num_boxes] containing
-                    weights for groundtruth boxes.
+        :param y: Targets of format `List[Dict[str, np.ndarray]]`, one for each input image. The fields of the Dict are
```

Comment:
I think the description of the format of `y` and  `standardise_output` is confusing here. What I understand from the code is that the format of `y` should depend on the value of `standardise_output`, so not fixed. But the description here seems that format of `y` is fixed as tensorflow and then will be converted inside to pytorch if `standardise_output` is True. So that's confusing.

Reply:
I agree.